src/Helius.ts

```diff
@@ -81,6 +82,12 @@ export class Helius {
     * @throws {Error} if there is an error calling the webhooks endpoint or if the response contains an error
     */
     async createWebhook(createWebhookRequest: CreateWebhookRequest): Promise<Webhook> {
+        // Filter out all account addresses that are on the ed25519 curve (valid Solana addresses)
+        const addressesOfCurve = createWebhookRequest.accountAddresses.filter(PublicKey.isOnCurve)
+        if (addressesOfCurve.length > 0) {
```

Comment:
we would wanna check that none of the keys are off the curve. This checks that any of the keys are on the curve

Reply:
Looks like I was drunk while doing tjese changes :D

I even have a typo in `addressesOfCurve`

I'll fix all the faulty stuff and have a proper commit history. Or discard this PR and let you guys handle fixes. Your call @vovkman

Reply:
thats fine, client side validation is good too